docs/3-building-blocks-of-oop-part-2/hometask/src/Weapon.ts

```diff
@@ -1 +1,67 @@
-// your code goes here
\ No newline at end of file
+import { Item } from "./Item";
+
+export class Weapon extends Item {
+  MODIFIER_CHANGE_RATE: number = 0.05;
+  baseDamage: number;
+  baseDurability: number;
+  damageModifier: number = 0;
+  durabilityModifier: number = 0;
+  effectiveDamage: number;
+  effectiveDurability: number;
+
```

Comment:
1. The first 4 properties should be private according to the UML diagram
2. The last 2 properties should be removed. Effective values should be calculated on the fly. Your approach will result in wrong logic.

Reply:
done